upgrade/helpers/tasks.py

```diff
@@ -148,12 +148,7 @@ def sync_capsule_repos_to_upgrade(capsules):
         # Fix of 1770940, 1773601
         logger.info("Resuming the cancelled content view {} publish task"
                     .format(cv.name))
-        output = run("sleep 100; hammer task list|awk '/Task canceled/{print $1}'; sleep 100")
```

Comment:
@devendra104 there is a foreman-maintain check to resume paused tasks. Should we give it try? and see if that works for us here.

Reply:
Thanks, Jameer for the review,  As we know,  foreman-maintain calls the same hammer command in the background to resume the task, I think to add one more layer is not good here.